vsts/linux_csharp_netcore.sh

```diff
@@ -2,12 +2,7 @@
 # Copyright (c) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-./build.sh -clean -configuration Debug
-if [ "$?" -ne 0 ]; then
-	exit 1
-fi
-
-./build.sh -clean -configuration Release -e2etests
+./build.sh -clean -build -configuration Debug -prtests
 if [ "$?" -ne 0 ]; then
```

Comment:
why not test it as Release?

Reply:
PR tests include unit tests, so they must be debug. Previously I was running unit tests in debug and then building for release and running the subset of E2E. By only doing debug in a PR we save another 90 seconds from building 2 flavors, which seemed like another nice gain.